.github/workflows/ci.yaml

```
@@ -263,11 +263,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: dtolnay/rust-toolchain@1.62.1
+      - uses: dtolnay/rust-toolchain@1.63.0
```

```suggestion
      - uses: dtolnay/rust-toolchain@stable
```
using stable is our common way

---

I applied the suggestion by amending [d60baae](https://github.com/RustPython/RustPython/pull/4051/commits/d60baaeab5a1aa0b44e8f2b9741a441a42a6f35c) commit.